### ISSUE TITLE
Change naming and describing required for timer and log

### ISSUE THREAD

**Issue by jnurthen** [w3c/aria-practices#1201]
Once https://github.com/w3c/aria/pull/1065 is merged then https://www.w3.org/TR/wai-aria-practices-1.1/#naming_role_guidance should be modified.

- I suggest `log` is modeled after `alert` as they appear similar
- for `timer` I really have no idea how a screen reader would ever read the accessible name... Anyone have any ideas as to what should be written here?
